incubator/appdaemon/0.1.0/test_values.yaml

```
@@ -0,0 +1,41 @@
+image:
+  repository: acockburn/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+
+env:
+  URL: ""
+  TOKEN: ""
+
+
+services:
+  main:
+    port:
+      port: 5050
+  tcp:
+    enabled: true
+    type: ClusterIP
+    port:
+      port: 51050
+      protocol: TCP
+      targetPort: 51050
+
```

This would be the correct definition for ingress:
```suggestion
ingress:
  main:
    enabled: false
```